iam/policies/vpc.tf

```diff
@@ -0,0 +1,146 @@
+# ------------------------------------------------------------------------------
+# Resources
+# ------------------------------------------------------------------------------
+resource "aws_iam_role_policy" "vpc" {
+  count  = "${contains(var.services, "ec2") && var.iam_role_name != "" ? 1 : 0}"
+  name   = "${var.prefix}-vpc-policy"
+  role   = "${var.iam_role_name}"
+  policy = "${data.aws_iam_policy_document.ec2.json}"
+}
+
+resource "aws_iam_user_policy" "vpc" {
+  count  = "${contains(var.services, "ec2") && var.iam_user_name != "" ? 1 : 0}"
+  name   = "${var.prefix}-vpc-policy"
+  user   = "${var.iam_role_name}"
+  policy = "${data.aws_iam_policy_document.vpc.json}"
+}
+
+data "aws_iam_policy_document" "vpc" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+                "ec2:AcceptVpcPeeringConnection",
+                "ec2:AllocateAddress",
+                "ec2:AssignPrivateIpAddresses",
+                "ec2:AssociateAddress",
+                "ec2:AssociateDhcpOptions",
+                "ec2:AssociateRouteTable",
+                "ec2:AttachClassicLinkVpc",
+                "ec2:AttachInternetGateway",
+                "ec2:AttachNetworkInterface",
+                "ec2:AttachVpnGateway",
+                "ec2:AuthorizeSecurityGroupEgress",
+                "ec2:AuthorizeSecurityGroupIngress",
+                "ec2:CreateCustomerGateway",
+                "ec2:CreateDhcpOptions",
+                "ec2:CreateFlowLogs",
+                "ec2:CreateInternetGateway",
+                "ec2:CreateNatGateway",
+                "ec2:CreateNetworkAcl",
+                "ec2:CreateNetworkAcl",
+                "ec2:CreateNetworkAclEntry",
+                "ec2:CreateNetworkInterface",
+                "ec2:CreateRoute",
+                "ec2:CreateRouteTable",
+                "ec2:CreateSecurityGroup",
```

Comment:
`ec2:CreateSecurityGroup` and other security group actions are already covererd by `ec2.tf`. We'll need a lot of overlaps in actions granted by the vpc policy and ec2, but I think we can still separate some concerns - e.g., I don't think someone should create a Security group unless they also intend to launch ec2 instances, so I think only the ec2 policy should grant security group privileges.